[CI] IpFieldBlockLoaderTests class failing

**Build Scans:**
- [elasticsearch-periodic-java-ea #334 / single-processor-node-tests](https://gradle-enterprise.elastic.co/s/3oudczfeyc2yi)
- [elasticsearch-pull-request #164748 / part-1](https://gradle-enterprise.elastic.co/s/y6jm4iivn25ie)
- [elasticsearch-pull-request #164748 / part-1](https://gradle-enterprise.elastic.co/s/fzmx5wfuom5su)
- [elasticsearch-pull-request #164010 / part-1](https://gradle-enterprise.elastic.co/s/hntrbulyy6oi6)
- [elasticsearch-pull-request #164010 / part-1](https://gradle-enterprise.elastic.co/s/2wdbsza327mlo)

**Reproduction Line:**
```
./gradlew ":server:test" --tests "org.elasticsearch.index.mapper.blockloader.IpFieldBlockLoaderTests.testBlockLoaderForFieldInObject {preference=Params[indexMode=columnar, sourceMode=COLUMNAR_STORED, preference=STORED]}" -Dtests.seed=51AB3323305401C2 -Dtests.configure_test_clusters_with_one_processor=true -Dtests.locale=rn-Latn-BI -Dtests.timezone=America/Tegucigalpa -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.index.mapper.blockloader.IpFieldBlockLoaderTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('org.elasticsearch.index.mapper.blockloader.IpFieldBlockLoaderTests'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: <[[0 0 0 0 0 0 0 0 0 0 ff ff 32 d a6 d1], [f b2 48 c6 ff ff ff ff 2 65 91 af ff ff ff ff], [7e 72 6f a4 ff ff ff ff ea 8e 83 c3 ff ff ff ff], [4e cd b2 fe ff ff ff ff 8c 84 76 e 0 0 0 0], [f0 20 f5 56 0 0 0 0 58 ca 3c f8 ff ff ff ff], [ba ae 3 63 0 0 0 0 46 ac f7 83 ff ff ff ff], [52 d9 ad a3 ff ff ff ff c5 e2 21 b8 ff ff ff ff], [dd 32 8c f2 ff ff ff ff 2e 7 3a 8a ff ff ff ff], [0 0 0 0 0 0 0 0 0 0 ff ff d7 3e a7 74], [0 0 0 0 0 0 0 0 0 0 ff ff 43 74 6c 8d], [0 0 0 0 0 0 0 0 0 0 ff ff a0 21 1c e6], [0 0 0 0 0 0 0 0 0 0 ff ff 27 ea 23 8], [df 2a bf 9e ff ff ff ff fc 11 6f be ff ff ff ff]]>
     but: was <[0 0 0 0 0 0 0 0 0 0 ff ff 32 d a6 d1]>
```

**Issue Reasons:**
- [main] 1 consecutive failures in class org.elasticsearch.index.mapper.blockloader.IpFieldBlockLoaderTests
- [main] 1 consecutive failures in step single-processor-node-tests
- [main] 1 consecutive failures in pipeline elasticsearch-periodic-java-ea
- [main] 5 failures in class org.elasticsearch.index.mapper.blockloader.IpFieldBlockLoaderTests (0.8% fail rate in 612 executions)
- [main] 4 failures in step part-1 (1.0% fail rate in 393 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (0.5% fail rate in 391 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -631,10 +631,6 @@ tests:
 - class: org.elasticsearch.xpack.dlm.frozen.DLMFrozenTransitionDisruptionIT
   method: testMasterFailoverDuringSnapshot
   issue: https://github.com/elastic/elasticsearch/issues/154395
-- class: org.elasticsearch.index.mapper.blockloader.IpFieldBlockLoaderTests
-  issue: https://github.com/elastic/elasticsearch/issues/154350
-- class: org.elasticsearch.index.mapper.blockloader.KeywordFieldBlockLoaderTests
-  issue: https://github.com/elastic/elasticsearch/issues/154349
 - class: org.elasticsearch.bootstrap.SpawnerNoBootstrapTests
   method: testControllerSpawnGatedBySettings
   issue: https://github.com/elastic/elasticsearch/issues/154291

--- a/test/framework/src/main/java/org/elasticsearch/datageneration/datasource/DefaultMappingParametersHandler.java
+++ b/test/framework/src/main/java/org/elasticsearch/datageneration/datasource/DefaultMappingParametersHandler.java
@@ -344,14 +344,13 @@ public class DefaultMappingParametersHandler implements DataSourceHandler {
             return ESTestCase.randomBoolean();
         }
 
-        // doc_values can't be disabled here.
+        // doc_values can't be disabled here; multi_value:false is exercised separately by SingleValueDocValuesDataSourceHandler.
         return ESTestCase.randomFrom(
             List.of(
                 true,
                 Map.of("multi_value", true),
                 Map.of("on_failure", ESTestCase.randomFrom("fail", "ignore")),
                 Map.of("multi_value", true, "on_failure", ESTestCase.randomFrom("fail", "ignore")),
-                Map.of("multi_value", false, "on_failure", "ignore"),
                 Map.of("nullability", false, "on_failure", "ignore")
             )
         );

